front/rule.common.php

```diff
@@ -147,5 +147,8 @@
     $rulecollection->menu_option
 );
 
-$rulecollection->display();
+$rulecollection->display([
+    'criterias' => 1,
+    'actions'   => 1,
```

Comment:
1. Boolean values seems more appropriate.
2. Names could be more meaningful.

```suggestion
    'display_criterias' => true,
    'display_actions'   => true,
```

Reply:
BTW, `criteria` is plural for `criterion`, please, no extra `s` ;p